### DISCUSSION TITLE
My profile picture changed without me knowing

### DISCUSSION THREAD

**Discussion by mkatch** [community/community#27272] [Other Feature Feedback, Questions, & Ideas] [answered]
A friend of mine just told me to check my GitHub profile because my picture features a woman in bikini. I have two-factor enabled and my password is used only in GitHub. I checked the security log and I don't see any suspicious activity. The only explanation I can think of is that it came from inside GitHub engineering. Does anyone have any idea what is going on? Is there a way to check when exactly the picture was changed?

<img width="453" src="https://user-images.githubusercontent.com/3180186/181214082-835e4415-7287-4773-a410-c48c2c2d3e46.png">

A reverse search shows some websites about bikinis. This is a public picture.

**Comment by jfullstackdev** [comment]
this is just an observation, coming from the API, your last profile update was 

<https://api.github.com/users/mkacz91>

`"updated_at": "2022-07-27T09:12:36Z"`

then, your latest commits 

![image](https://user-images.githubusercontent.com/47092464/181216843-ab4fdcb4-c1bd-482e-9e89-859cd8288fde.png)

but it does not appear at the timeline, 

![image](https://user-images.githubusercontent.com/47092464/181216974-71b83289-d5b5-4776-adc5-bd41edb00e15.png)

and they are unverified commits

![image](https://user-images.githubusercontent.com/47092464/181217183-8924692b-4508-4f0f-bb6b-fc057e6ca49b.png)

seems it's running GitHub Actions ?

**Reply by mkatch** [reply]
I am using GitHub Actions indeed, and I don't see any suspicious activity there.

The last update time is shortly after I received the message from a friend and visited my profile. I'm not sure what I changed then, maybe I changed something while going through various settings trying to figure out what happened. But since the API only shows the last alteration timestamp, then the previous one is lost.

**Comment by rowan-mcalpin** [comment]
Additionally, why is it stretched out like that? I've never seen that before...

**Comment by ettaboyle** [comment, marked as answer]
Hi @mkacz91 👋🏻  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

[Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
* [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
* [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
* [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

 You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
